common/src/main/java/com/revenuecat/purchases/common/Config.kt

```diff
@@ -1,8 +1,10 @@
 package com.revenuecat.purchases.common
 
+import com.revenuecat.purchases.LogLevel
+
 object Config {
 
-    var debugLogsEnabled = BuildConfig.DEBUG
+    var logLevel: LogLevel = LogLevel.debugLogsEnabled(BuildConfig.DEBUG)
```

Comment:
I believe you don't need to specify the type here
```suggestion
    var logLevel = LogLevel.debugLogsEnabled(BuildConfig.DEBUG)
```

Reply:
Oh thanks! I write very little Kotlin so please keep the picky comments coming.